build/noderesourcetopology-plugin/Dockerfile.tools

```diff
@@ -17,7 +17,6 @@ RUN yum --setopt=install_weak_deps=False -y install \
     gettext \
     which \
     findutils \
-    python2 \
```

Comment:
better to move this commit to its own PR (which will still hit broken CI) and force-merge it. Then this PR should pass CI